### ISSUE TITLE
JOJ Play

### ISSUE THREAD

**Issue by marki555** [kodi-czsk/plugin.video.joj.sk#50]
Vyzera to tak ze JOJ zacala presuvat svoj archiv pod portal JOJ Play, minimalne serial Nemocnica sa uz pod archivom pozriet neda a odkazuje na JOJ Play, ktore vyzaduje registraciu (ale je aj Free verzia bez platenia).
Je v plane podpora aj tohto portalu? Pripadne uz existuje / je vo vyvoji nejaky iny doplnok pre JOJ Play?

**Comment by Ewors** [comment]
> Vyzera to tak ze JOJ zacala presuvat svoj archiv pod portal JOJ Play, minimalne serial Nemocnica sa uz pod archivom pozriet neda a odkazuje na JOJ Play, ktore vyzaduje registraciu (ale je aj Free verzia bez platenia). Je v plane podpora aj tohto portalu? Pripadne uz existuje / je vo vyvoji nejaky iny doplnok pre JOJ Play?

Konečne niekto kto sa nato opýtal. Už som si myslel ze je chyba vo mne. O joj play som ani nevedel.

**Comment by jose1711** [comment]
@Ewors a to si sa bal opytat sam? Mimochodom riesilo sa to aj tu: https://github.com/kodi-czsk/plugin.video.joj.sk/issues/34

**Comment by Ewors** [comment]
> @Ewors a to si sa bal opytat sam? Mimochodom riesilo sa to aj tu: #34

Nie. Len až dnes má to napadlo hodiť do Google a nájsť toto git vlákno. Prvé čo ma napadlo bolo pozrieť na xbmc fórum kde to nikto neriešil.

**Comment by filip-769** [comment]
Spravil som doplnok pre Joj Play: https://github.com/filip-769/plugin.video.play.joj.sk
